README.md: Add `docker build` from repo URL (#32)
orhun/pkgtop: README.md

```
docker build -f docker/Dockerfile -t pkgtop-docker .
```

> or, if you don't want to clone the repo, just do

Reviewer: ```suggestion
or if you don't want to clone the repository, you can run:

```
